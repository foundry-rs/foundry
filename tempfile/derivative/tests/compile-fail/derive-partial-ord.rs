#[macro_use]
extern crate derivative;

#[derive(Derivative, PartialEq)]
#[derivative(PartialOrd)]
enum Option {
    Some,
    None,
}

fn main() {}
