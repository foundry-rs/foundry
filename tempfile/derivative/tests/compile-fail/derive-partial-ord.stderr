error: can't use `#[derivative(PartialOrd)]` on an enumeration without `feature_allow_slow_enum`; see the documentation for more details
 --> $DIR/derive-partial-ord.rs:5:1
  |
5 | #[derivative(PartialOrd)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^
